src/pages/EditMonitor.vue

```diff
@@ -377,20 +377,11 @@
                             <template v-if="monitor.type === 'sqlserver' || monitor.type === 'postgres' || monitor.type === 'mysql'">
                                 <div class="my-3">
                                     <label for="sqlConnectionString" class="form-label">{{ $t("Connection String") }}</label>
-
-                                    <template v-if="monitor.type === 'sqlserver'">
-                                        <input id="sqlConnectionString" v-model="monitor.databaseConnectionString" type="text" class="form-control">
-                                    </template>
-                                    <template v-if="monitor.type === 'postgres'">
-                                        <input id="sqlConnectionString" v-model="monitor.databaseConnectionString" type="text" class="form-control">
-                                    </template>
-                                    <template v-if="monitor.type === 'mysql'">
-                                        <input id="sqlConnectionString" v-model="monitor.databaseConnectionString" type="text" class="form-control">
-                                    </template>
+                                    <input id="sqlConnectionString" v-model="monitor.databaseConnectionString" type="text" class="form-control" required>
                                 </div>
                                 <div class="my-3">
                                     <label for="sqlQuery" class="form-label">{{ $t("Query") }}</label>
-                                    <textarea id="sqlQuery" v-model="monitor.databaseQuery" class="form-control" placeholder="Example: select getdate()"></textarea>
+                                    <textarea id="sqlQuery" v-model="monitor.databaseQuery" class="form-control" :placeholder="$t('sqlQueryExample: select getdate()')" required></textarea>
```

Comment:
Hi

When I was doing https://github.com/louislam/uptime-kuma/pull/3439 I've noticed that some DB monitoring implementations supported a default monitoring query. 

I found this to be really nice because I'm all for "Convention over Configuration" 😄 as it makes it easier for a new user to setup the DB monitoring. Still, I agree with you that, right now, it is not really clear for the user if the query is mandatory or not.

IMHO I think that having a default query for each DB monitoring would be interesting. Yet, it would probably be better to have some visual queue so the user understands if  the query is optional and knows what's the default query for the selected DBMS.

What do you think?

Reply:
I could not find such a query for postgres, mysql or sqlserver.
Having default queries could be a thing that would be nice, but I think this would need to be a different PR.

Reply:
> I could not find such a query for postgres, mysql or sqlserver.

Yeah, it's not really clear unless one digs onto the `util-server.js`:
```js
/**
 * Run a query on Postgres
 * @param {string} connectionString The database connection string
 * @param {string} query The query to validate the database with
 * @returns {Promise<(string[]|Object[]|Object)>}
 */
exports.postgresQuery = function (connectionString, query) {
    return new Promise((resolve, reject) => {
        const config = postgresConParse(connectionString);

        if (config.password === "") {
            // See https://github.com/brianc/node-postgres/issues/1927
            return reject(new Error("Password is undefined."));
        }

        const client = new Client({ connectionString });

        client.connect((err) => {
            if (err) {
                reject(err);
                client.end();
            } else {
                // Connected here
                try {
                    // No query provided by user, use SELECT 1
                    if (!query || (typeof query === "string" && query.trim() === "")) {
                        query = "SELECT 1";
                    }
...
```

I liked the idea and this the same on my PR (https://github.com/louislam/uptime-kuma/pull/3439), but it won't probably be merged because DB2 doesn't offer a native JS module to interface with the DB 😞 

> Having default queries could be a thing that would be nice, but I think this would need to be a different PR.
Agreed 👍